pkg/api/server/cel2sql/convert_test.go

```diff
@@ -44,6 +45,16 @@ func TestConvertRecordExpressions(t *testing.T) {
 			in:   `data.metadata.labels["foo"] == "bar"`,
 			want: "(data->'metadata'->'labels'->>'foo') = 'bar'",
 		},
+		{
+			name: "concatenate strings",
+			in:   `name + "bar" == "foobar"`,
+			want: "CONCAT(name, 'bar') = 'foobar'",
```

Comment:
I implemented first using the operator `||`, but then I read the following:
```
One disadvantage of using the || operator is a null value in any of the columns being joined together will result in a null value.
```
So I decided to implement using CONCAT function, which it's present in PostgreSQL and MySQL. But then the test fails because it's not present in SQLite. Other databases like SQL Server only accept `+`.

Not exactly sure how to follow 🤔

Reply:
I want to make the interpreter configurable so we can easily change the string concat strategy. WDYT?